src/security/README.md

```diff
@@ -1,3 +1,82 @@
-# Security (TO BE UPDATED)
+# Data Plane Security Module
 
-Details on certificate management and process authentication by the agent are available [here](/src/control/security/README.md).
+The DAOS security module centralizes all access and security-related
+functionality in the DAOS data plane in a single module.
+
+The functionality in this module includes:
+
+- Client library requests to the DAOS Agent for a credential for authentication.
+- Server requests to the Control Plane to validate a signed client credential.
+- Generating default Access Control Lists for pools and containers.
+- Deriving pool and container capabilities from the combination of the Access
+  Control List and the client credential.
+- Access control checks for pool and container operations.
+
+## Credential Generation and Validation
+
+Details on the credential generation and validation process
+are outlined
+[in the Control Plane documentation](/src/control/security/README.md).
+
+## Access Control Internals
+
+This section covers the implementation details of pool and container access
+control.
+
+See the [ACL section of the User Guide](/doc/user/acl.md) for background on DAOS
+Access Control Lists.
+
+See the [Admin Guide](/doc/admin/pool_operations.md#pool-access-control-lists)
+for a higher-level view of pool access control.
+
+See the [User Guide](/doc/user/container.md#container-access-control-lists)
+for a higher-level view of container access control.
+
+### Pool Access
+
+When a client connects to a pool, the user credential that was used to make the
+connection is used, in combination with the pool ACL and the requested access
+type (RO or RW), to generate an internal set of security capabilities for the
+pool handle. These capabilities map to the valid permissions set for pools and
+are used for future access decisions for that pool handle, for pool-level
+operations such as container creation.
+
+These capabilities persist for the life of the handle, even if someone modifies
+the pool ACL or ownership. The modifications won't be used for access decisions
+until the user attempts to get a new handle for the pool. The pool handle cannot
+be revoked.
+
+In addition, the validated credential is saved in the handle data in the DAOS
+data plane during pool connect, and remains associated with the pool handle
+for its lifetime.
+This original credential is used for access decisions when the user
```

Comment:
This either needs to be merged into the previous line or have another line added if it is supposed to be its own paragraph.

Reply:
Ah, must have missed it when I was trying to clean up the paragraph widths. It is supposed to be part of the preceding paragraph. In the final rendering it looks like it does the right thing.